CHANGES.md

```diff
@@ -14,6 +14,7 @@ Change Log
 * `CesiumTerrainProvider` now supports terrain tiles using a `WebMercatorTilingScheme` by specifying `"projection": "EPSG:3857"` in `layer.json`. It also now supports numbering tiles from the North instead of the South by specifying `"scheme": "slippyMap"` in `layer.json`. [#8563](https://github.com/CesiumGS/cesium/pull/8563)
 * Added basic support for `isNaN`, `isFinite`, `null`, and `undefined` in the 3D Tiles styling GLSL backend for point clouds. [#8621](https://github.com/CesiumGS/cesium/pull/8621)
 * Updated knockout from 3.5.0 to 3.5.1.
+* Added `RequestScheduler` to public API. [#8384](https://github.com/AnalyticalGraphicsInc/cesium/issues/8384)
```

Comment:
This should be a little more of a description of what the `RequestScheduler` actually is